root/artist/ArtistMerge.js

```diff
@@ -65,9 +65,23 @@ const ArtistMerge = ({$c, form, toMerge}: Props) => {
           <FormRowCheckbox
             field={form.field.rename}
             label={l(`Update matching artist and relationship credits to use
-                      the new artist’s name`)}
+                      the target artist’s name`)}
           />
 
+          <p>
+            {exp.l(
+              `By default, the merge will combine the artists but the previous
+              name(s) will still be used on the {doc_acs|artist credits}. If
+              all the name(s) being merged are incorrect (e.g. typos) and all
+              existing credits matching those names should be updated, you can
+              click the checkbox above to do so. Keep in mind artist credits should
+              generally follow what is printed on releases. If an artist has
+              releases using several of the names being merged, do not change the
```

Comment:
```suggestion
               releases using any of the names being merged, do not change the
```
(Otherwise it's saying it's okay to update all of the names if only some are unused, right?)

Reply:
I think I meant "several" as in including the one being merged *into*, but you're right this'd make it clearer.